### PR TITLE
[TTAHUB-571] Cleanup AR Landing with New Filters

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

Cleanup for AR Landing with new filters. 

Fix issues found in UI review.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-571

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/pages/Landing/index.css [hunk 1]
```diff
@@ -250,14 +250,10 @@ a.desc {
   background: #c5c5c5;
 }
 
-#caughtUp {
```
Comment (thewatermethod) [inline frontend/src/pages/Landing/index.css:253]: You could remove the ```.smart-hub--caught-up``` class entirely and replace it with helper classes. I don't think you can match padding exactly but those values look fairly arbitrary. Also don't think we need ```white-space: nowrap;```. I'd try removing that class and this css and replacing it with like ```text-center padding-10``` or something like that.

File: frontend/src/pages/Landing/index.css [hunk 2]
```diff
@@ -250,14 +250,10 @@ a.desc {
...
   white-space: nowrap;
-}
-
-#caughtUp h2 {
-  white-space: nowrap;
-  padding-left: 50px;
+  text-align: center;
 }
 
 #beginNew {
```
Comment (thewatermethod) [inline frontend/src/pages/Landing/index.css:259]: This ```#beginNew``` can probably also just be replaced with a helper class on whichever element it applies to

File: frontend/src/pages/Landing/index.css (hunk 1 above)
Comment (AdamAdHocTeam) [reply frontend/src/pages/Landing/index.css:253]: I have updated the CSS and used the helper class.

File: frontend/src/pages/Landing/index.css (hunk 2 above)
Comment (AdamAdHocTeam) [reply frontend/src/pages/Landing/index.css]: CSS cleaned up and helper class applied.

File: frontend/src/pages/Landing/MyAlerts.js [hunk 3]
```diff
@@ -247,12 +247,12 @@ function MyAlerts(props) {
     <>
       {reports && reports.length === 0 && (
         <Container className="landing" padding={0} loading={loading}>
-          <div id="caughtUp">
+          <div className="text-center padding-10">
             <div>
-              <h2>You&apos;re all caught up!</h2>
+              <h2 className="padding-left-0">You&apos;re all caught up!</h2>
```
Comment (thewatermethod) [inline frontend/src/pages/Landing/MyAlerts.js:252]: this doesn't need ```padding-left-0```

**Review by thewatermethod — APPROVED** [review]
👍

**Pull request merged**